packages/peregrine/lib/store/actions/user/asyncActions.js

```diff
@@ -7,8 +7,11 @@ import actions from './actions';
 
 const storage = new BrowserPersistence();
 
-export const signOut = ({ history }) => async dispatch => {
-    // Sign the user out in local storage and Redux.
+export const signOut = ({ history, revokeToken }) => async dispatch => {
+    // Send mutation to revoke token.
+    await revokeToken();
```

Comment:
Technically this will return an error if called without a bearer token. Do we care? Probably not, but I would possibly catch and do something with the error just incase.

```json
{
  "errors": [
    {
      "message": "The current customer isn't authorized.",
      "category": "graphql-authorization",
      "locations": [
        {
          "line": 2,
          "column": 5
        }
      ],
      "path": [
        "revokeCustomerToken"
      ]
    }
  ],
  "data": {
    "revokeCustomerToken": null
  }
}
```

Reply:
Fixed in 1f680c4